public/docs/_examples/dependency-injection/dart/lib/app_component_1.dart

```diff
@@ -1,17 +1,17 @@
 // Early versions
 
 // #docregion
-import 'package:angular2/core.dart';
+import 'package:angular2/angular2.dart';
```

Comment:
@zoechi Would you be able to dig up that recommendation? Would be very interested to see the rationale.